backend/pipeline.yaml

```diff
@@ -1,18 +1,19 @@
+---
 $schema: "pipeline.schema.v1"
 serviceGroup: Microsoft.Azure.ARO.HCP.RP.Backend
 rolloutName: RP Backend Rollout
 resourceGroups:
-- name: {{ .svc.rg }}
-  subscription: {{ .svc.subscription }}
-  aksCluster: {{ .aksName }}
+- name: {? {.svc.rg: ''} : ''}
```

Comment:
Are these changes with go templates conditional/ternary operators expected as part of this PR?

Reply:
This also changes the behavior where instead of the template failing if the variable is not set, it'll set defaults to empty strings which may not produce desired outcome. Seems like its better to let it fail than to use empty strings and hope for the best?

Reply:
the templates fail and should fail if a variable can't be resolved

Reply:
The yaml formatter changed this for us.  I think we have to convert things to strings as a result.

Reply:
Oh wow TIL https://yaml.org/spec/1.2.2/#rule-e-node
I didn't think this was valid YAML... To essentially this not only add some yaml ternary blocks but breaks our templating?

Is there a rule we can exclude from the formatter to avoid it messing with our templating?